### ISSUE TITLE
Can you provide a source?

### ISSUE THREAD

**Issue by gregmcinnes** [JHart96/keras_ordinal_categorical_crossentropy#1]
Where is the equation you use for ordinal categorical crossentropy derived from?  Is there a publication you can point to?

Thanks

**Comment by JHart96** [comment]
I came up with the equation myself as a solution to a problem I was working on, as such I'm afraid there is no paper I can link to. I've analysed it on my use case (fine-grained sentiment analysis) and found that it achieves 1.96% fewer misclassifications more than 1 class away from the true class.

Here are the normalized confusion matrices for categorical crossentropy and ordinal categorical crossentropy from my analysis:

![image](https://user-images.githubusercontent.com/6945724/45368154-d96b8380-b5da-11e8-85e2-1cc555d700fc.png)

![image](https://user-images.githubusercontent.com/6945724/45368229-fd2ec980-b5da-11e8-9fe5-7d5772f2fb8d.png)

I hope this helps.

Thanks,

Jordan
